core-processor/src/main/java/io/micronaut/expressions/context/ExpressionContextReference.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.expressions.context;
+
+import io.micronaut.core.annotation.Internal;
+
+/**
+ * A reference for expression context class.
+ *
+ * @author Sergey Gavrilov
+ * @since 4.0.0
+ */
+@Internal
+public abstract class ExpressionContextReference {
```

Comment:
It can't. This references are only generated at compilation time and written to META-INF to let other modules discrover classes annotated with `@EvaluatedExpressionContext` which were compiled separately